.eslintrc.js

```diff
@@ -145,10 +145,9 @@ function styleRules(level) {
     }],
     '@stylistic/jsx-indent-props': [level, 2],
     '@stylistic/jsx-max-props-per-line': [level, {
-      maximum: 1,
+      maximum: 3,
       when: 'multiline',
     }],
-    '@stylistic/jsx-one-expression-per-line': [level, { allow: 'single-child' }],
```

Comment:
This was warning on `{rect} {x.name}: {pctText}` and was asking me to have one per line, which seemed excessive